kpi/README.rst

```diff
@@ -0,0 +1,78 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Can you divide the readme following the new rules?
https://github.com/OCA/maintainer-tools/tree/master/template/module/readme